lib/gitsh/environment.rb

```diff
@@ -88,7 +87,7 @@ def repo_has_untracked_files?
     end
 
     def repo_config_color(name, default)
-      if color_override = self[name]
+      if color_override = fetch(name) { nil }
```

Comment:
Interesting - so how is this better than what we had before? (I previously understood the benefit of `fetch` to be that it would not return `nil`)